gui/default/syncthing/core/syncthingController.js

```diff
@@ -455,6 +486,16 @@ angular.module('syncthing.core')
             }
         }
 
+        function refreshCluster() {
+            $http.get(urlbase + '/cluster/pendingdevices').success(function (data) {
```

Comment:
/cluster/pending/foo in case we add something else pending in the future?

Reply:
I don't see what else could be pending about the *cluster*.  Pending notifications maybe, but that's not relevant to the cluster.  I don't really care, but in fact it does stand out for being the only two-word endpoint.

Should I change it?

Reply:
Yes, if you could, please.

Reply:
Fixed, tested and pushed.